guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -25,38 +25,44 @@ include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[le
 endif::[]
 
 // Upgrading Satellite Server overview
-include::topics/upgrading_overview.adoc[leveloffset=+1]
+include::topics/con_upgrading-overview.adoc[leveloffset=+1]
 
-include::topics/upgrading_capsules_separately_from_satellite.adoc[leveloffset=+2]
+include::topics/con_upgrading-prerequisites.adoc[leveloffset=+1]
+
+include::topics/con_upgrade-paths.adoc[leveloffset=+1]
+
+include::topics/con_following-the-progress-of-the-upgrade.adoc[leveloffset=+1]
+
+include::topics/con_upgrading-capsules-separately-from-satellite.adoc[leveloffset=+2]
 
 [#upgrading_{project-context}]
 == Upgrading {ProjectName}
 
 // Upgrading Red Hat Satellite Introduction
-include::topics/introduction_upgrading_satellite.adoc[leveloffset=+2]
+include::topics/con_introduction-upgrading-satellite.adoc[leveloffset=+2]
 
 // Upgrading Satellite Server
-include::topics/upgrading_satellite_server_parent.adoc[leveloffset=+2]
+include::topics/con_upgrading-satellite-server-parent.adoc[leveloffset=+2]
```

Comment:
```suggestion
include::topics/con_satellite-server-upgrade-scenarios-and-prerequisites.adoc[leveloffset=+2]
```
I'd rename this file to match the content and upsize the following modules in the TOC instead of keeping it a "parent" to flatten the structure.